extensions/positron-r/src/provider.ts

```diff
@@ -293,58 +293,60 @@ function binFragments(): string[] {
 }
 
 export async function findCurrentRBinary(): Promise<string | undefined> {
-	if (os.platform() === 'win32') {
-		const registryBinary = findCurrentRBinaryFromRegistry();
-		if (registryBinary) {
-			return registryBinary;
-		}
-	}
+	// if (os.platform() === 'win32') {
+	// 	const registryBinary = findCurrentRBinaryFromRegistry();
+	// 	if (registryBinary) {
+	// 		return registryBinary;
+	// 	}
+	// }
```

Comment:
Just a temporary measure, so I can actually exercise the PATH code.

I ?think? the registry should take precedence over PATH re: Positron's understanding of the default version of R. Because it is a much more official way of designating the default on Windows, i.e. it's the only thing the official installer is willing to do for you.

But if someone has a different opinion, let's discuss.